test/unit/vl-pager.test.html

```diff
@@ -209,6 +209,21 @@
         pager.setAttribute('pagination-disabled', '');
         should.not.exist(pager._pagesElement);
       });
+
+      test('er wordt een change event gegooid telkens huidige pagina verandert', () => {
+        const pager = fixture('vl-pager-fixture');
+        const receivedEvents = [];
+        pager.addEventListener('change', e => {
+        	receivedEvents.push(e.detail);
+        });
+        pager.shadowRoot.querySelector("#page-forward-link").click();
+		assert.deepEqual(receivedEvents[receivedEvents.length -1], {currentPage: "2", totalPage: 10, itemsPerPage: 10, totalItems: 100});
```

Comment:
Gaat dit niet flaky zijn? We wachten hier niet tot we zeker zijn dat de events verwerkt zijn.

Reply:
toch synchroon?

Reply:
Veiliger is inderdaad om in de event listener pas de volgende test code te zetten.

Reply:
> toch synchroon?

Ik denk niet dat je er van uit mag gaan dat een browser een event meteen afhandelt. Het zou kunnen zijn dat de browser de event pas de volgende 'tick' verwerkt.